### PR TITLE
Improve Angular Integration

### PR DESCRIPTION
**Pull request by ibirrer**
This PR focuses on the integration of the six-webcomponents with Angular, in particular for Angular forms. It also removes [browser built in form validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#using_built-in_form_validation) to reduce the complexity of the library. Details below.

## Angular
* Built-In Angular form support
  * [Value accessors](https://angular.io/api/forms/ControlValueAccessor#controlvalueaccessor) for all form elements
  * Validators for datepicker
  * Support for min/max validation in `six-input[type=number]`
* Default error messages for basic validations for all elements in English, German, Italian and French. 
* Ability to customize default messages
* Support to mark all form inputs as touched and dirty and focus first invalid field (`SixFormUtilDirective`)
* Simplified Angular example
  * Removed ngrx
  * Removed table and user from
  * Added form example demoing all form field types and error message customization

## Other
* Added `input, blur, change, focus` events to all form elements as counterparts/replacements for custom events like `six-input-input, six--input-change` etc. Allows for better integration with third party form libraries that rely on those native events.
* Support submitting a form by pressing the 'Enter' key on six-input.
* Removed Vue Example and stencil output target for Vue. 
* Removed support for [browser built in form validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation#using_built-in_form_validation).
  * Removed `reportValidity(), checkValidity(), setCustomValidity(), getValidity(), isValid(), getValidationMessage()
  * Removed `reset(), defaultValue`
  * Removed `six-form element`
* Removed support for property `customErrorText`. Use `errorText` instead
* Removed support for `error-text` slot. Use the `errorText` property instead
* Removed `errorOnBlur` property

## Upgrade
We want to provide an upgrade guide after merging this PR. For the moment, check the angular example which contains a new form example page: https://github.com/six-group/six-webcomponents/tree/feature/remove-native-validation/examples/angular/src/app/pages/form

In general, all you need to to is to import the Angular module `UiLibraryAngularModule.forRoot()` and write your Angular forms as usual.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,6 @@
     "build:lib": "npm run build -w libraries",
     "lint:lib": "npm run lint -w libraries --if-present",
     "test:unit": "npm run test -w libraries/ui-library",
-    "demo:vue": "npm start -w examples/vue",
```
Comment (colinscz) [inline package.json:9]: @pennal FYI I suggest you do a rebase after this is merged, probably you have some conflicts with your fork

**Review by colinscz — COMMENTED** [review]
First round of review through, will take another look. Just 1 minor question so far :)

File: libraries/ui-library-angular/package.json [hunk 2]
```diff
@@ -8,18 +8,18 @@
     "watch": "ng build --watch --configuration development"
   },
   "peerDependencies": {
-    "@angular/common": "^15.2.0",
-    "@angular/core": "^15.2.0",
+    "@angular/common": ">=14.0.0",
```
[the remark this answers is not in the document]
Comment (ibirrer) [reply libraries/ui-library-angular/package.json:11]: This defines the minimum version you need in your application to use the library (peer dependency). The dependencies below set the versions that this library is being built with.

**Review by colinscz — APPROVED** [review]
After review discussion with Iwan and pair- review ready for merge

**Pull request merged**